Documentation/gettingstarted/clustermesh/services.rst

```diff
@@ -110,3 +142,61 @@ Deploying a Simple Example Service
       kubectl exec -ti deployment/x-wing -- curl rebel-base
 
    You will see replies from pods in both clusters again.
+
+9. In cluster 1, add ``io.cilium/service-affinity="local"`` to existing global service
+
+   .. code-block:: shell-session
+
+      kubectl annotate service rebel-base io.cilium/service-affinity="local" --overwrite
+
+10. From cluster 1, access the global service:
```

Comment:
nit: If each of these numbers is `1.` and the content inside each of these numbered bullets is indented starting from the same indentation as the non-space character following the `1.`, then sphinx should autonumber these for you. :sweat_smile: 

(But I'm also okay with just writing all the numbers out like this if it's easier).

Reply:
good tip :100: , but I keep the running sequence to be consistent with other pages.